frontend/packages/eslint-plugin-console/lib/config/rules/typescript.js

```diff
@@ -74,8 +74,17 @@ module.exports = {
   //   },
   // ],
 
+  // TODO This rule wasn't working as expected prior to @typescript-eslint/eslint-plugin v2.11.0.
+  // Disabled for now as it is causing failures
   // Enforces the use of as Type assertions instead of <Type> assertions
-  '@typescript-eslint/no-angle-bracket-type-assertion': 'error',
+  // '@typescript-eslint/consistent-type-assertions': [
+  //   'error',
+  //   {
+  //     assertionStyle: 'as',
```

Comment:
If I recall, this is recommended, but someone would likely have to have real strong feelings to comb the code & update :)

Reply:
Ya we definitely want this and we'll have to address whatever issue causes this to not work properly.